functions/common/function-test-support/src/main/resources/docker-compose/xmpp/docker-compose.yml

```diff
@@ -1,6 +1,6 @@
 version: "3.7"
 services:
-  Openfire:
+  xmpp:
     image: fishbowler/openfire:latest
```

Comment:
It may be less surprised down the road if we lock onto a specific version (`v4.7.0` seems to be a good candidate).